helpers.ts

```diff
@@ -276,6 +276,20 @@ export function connectEventually(factory: () => net.Socket, handler: (_socket:
 	tryConnect();
 }
 
+export function convertToDictionary(arr: string[]): IStringDictionary {
+	let dictionary: IStringDictionary = Object.create(null);
+	_.each(arr, item => {
+		let [key, value] = item.split(" ");
```

Comment:
The file name may contain spaces and this code will fail in such case. We actually defend on other places in this PR against this by using double quotes, like https://github.com/telerik/mobile-cli-lib/pull/591/files#diff-1596e63db9f2f8338a52444248dee614R68 .